Timefy/View/CurrentTaskComponent/CurrentTaskComponent.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  CurrentTaskComponent.swift
+//  Timefy
+//
+//  Created by Doğancan Mavideniz on 12.09.2022.
+//
+
+import UIKit
+
+class CurrentTaskComponent: UIView {
+
+    static let identifier = "CurrentTaskComponent"
+
+    @IBOutlet private weak var timeTextLabel: UILabel!
+    @IBOutlet private weak var projectNameLabel: UILabel!
+    @IBOutlet private weak var goDetailButton: UIButton!
+    @IBOutlet private weak var workTypeIconImage: UIImageView!
+    required init?(coder aDecoder: NSCoder) {
```

Comment:
outlet ile init arası boşluk olsa daha derli toplu olabilir. 11,13'te bırakıp 18'e gelince boşluk koymaman için seni ne durdurdu?